packages/devtools_app/lib/src/config_specific/flutter/drag_and_drop/drag_and_drop.dart

```diff
@@ -0,0 +1,6 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+export '_drag_and_drop_desktop.dart'
```

Comment:
I would use conditional imports here instead of conditional imports.
using conditional exports is a last resort and the analyzer won't do a good job of surfacing bugs if you do it that way. I think you can expose the same functionality you want using conditional imports instead.

With conditional imports, I would suggest the following:
Rename 
DragAndDropBaseState --> DragAndDropState